README.md

```diff
@@ -93,6 +93,10 @@ To get a local development version of `0x-api` running:
 | `META_TXN_RELAY_EXPECTED_MINED_SEC`    | Default: `120`                                                  | The expected time for a meta-txn to be included in a block.                                                                                                                                               |
 | `ENABLE_PROMETHEUS_METRICS`            | Default: `false`                                                | A boolean determining whether to enable prometheus monitoring.                                                                                                                                            |
 | `PROMETHEUS_PORT`                      | Default: `8080`                                                 | The port from which prometheus metrics should be served.                                                                                                                                                  |
+| `LOG_LEVEL`                            | Default: `info`                                                 | The log level for `pino.js`.                                                                                                                                                                              |
+| `LOG_ASYNC_ENABLED`                    | Default: `false`                                                | A boolean determining whether or not to enable async logging.                                                                                                                                             |
+| `LOG_BUFFER_BYTES`                     | Default: `0`                                                    | The log write buffer size.                                                                                                                                                                                |
+| `LOG_ASYNC_FLUSH_INTERVAL_SEC`         | Default: `10`                                                   | The log flush interval in seconds when `LOG_ASYNC_ENABLED` is set to true.                                                                                                                                |
```

Comment:
So logs could be delayed by up to 10 seconds? That's fine for me, but is there any process or alarm that depends on data being available much faster?

Reply:
For the logs to be delayed:
- `sync` has to be set to `false` (enabling async logging)
- there has to be minimal activity to not fill up the given `LOG_BUFFER` (for example `4kb`) and flush automatically.
- The external services that depend on timing can use the `pino`'s timestamp contained in the log object, not on the flush time.